compat/sphinxcontrib_verilog_diagrams/__init__.py

```diff
@@ -0,0 +1,11 @@
+import sphinxcontrib_hdl_diagrams
+
+def setup(app):
+    print("")
```

Comment:
Use logging.warning -- http://www.jaggedverge.com/2016/09/deprecation-warnings-in-python/

Reply:
Done with `warnings.warn` as in the article that you presented